src/lifecycle.mjs

```diff
@@ -8,30 +8,21 @@ import EventEmitter from "events";
 import { env, exit } from "process";
 
 import Ajv from "ajv";
-import partition from "lodash.partition";
 import { lifecycleMessage } from "@neume-network/message-schema";
 
-import {
-  NotImplementedError,
-  NotFoundError,
-  ValidationError,
-} from "./errors.mjs";
+import { NotFoundError, ValidationError } from "./errors.mjs";
 import { loadStrategies, write } from "./disc.mjs";
 import logger from "./logger.mjs";
 
 const log = logger("lifecycle");
-const __dirname = path.dirname(fileURLToPath(import.meta.url));
 const strategyDir = "./strategies";
 // TODO: https://github.com/neume-network/core/issues/33
-const dataDir = path.resolve(__dirname, "../../..", env.DATA_DIR);
```

Comment:
I don't understand this change, is it correct?

Reply:
This change was prompted because the CLI can be used from any directory and not just the neume-network/core directory. `path.resolve` will return an absolute path from where my code is being run. 

For eg, if I am in `/users/vaibhav` and run `neume --data-dir=data` then I want the data directory to be created at `/users/vaibhav/data` and not relative to where `neume` is stored.

Reply:
ok thx for explaining